rotkehlchen/api/rest.py

```diff
@@ -645,6 +645,12 @@ def query_exchange_balances(
 
         return api_response(_wrap_in_ok_result(process_result(balances)), HTTPStatus.OK)
 
+    def get_supported_chains(self) -> Response:
+        result = []
+        for entry in SupportedBlockchain:
+            result.append({'name': entry.value, 'type': entry.get_chain_type()})
```

Comment:
Is there a reason to not just do? Asking because is the classic example for list comprehension but if is style is fine. Just wanted to know

```suggestion
        result = ['name': entry.value, 'type': entry.get_chain_type()} for entry in SupportedBlockchain]
```

Reply:
no reason. Just started the one kelsos asked and then changed it to the other one so did not do a comprehensions